R/testthat-ui.R

```diff
@@ -131,22 +131,26 @@ check_versions_match <- function(dep, system_ver, strip_minor = FALSE) {
     )
     return_from(caller_env(), skipped_mismatch_exp(msg, case))
   }
-
-  if (cases_ver < system_ver) {
+  else if (cases_ver < system_ver) {
     msg <- glue(
       "Failed doppelganger was generated with an older { dep } version.
        Please revalidate cases with vdiffr::validate_cases() or vdiffr::manage_cases()"
     )
     return_from(caller_env(), skipped_mismatch_exp(msg, case))
   }
-
-  if (cases_ver > system_ver) {
+  else if (cases_ver > system_ver) {
```

Comment:
I don't think these `else` are needed, could you remove them please?

Reply:
Easy enough.